api/src/gotEnzymes/enzyme.js

```diff
@@ -9,6 +9,7 @@ const MATCH_FIELDS = [
   'compound',
 ];
 
+const CASED_FIELDS = ['gene', 'organism', 'domain', 'compound', 'reaction_id'];
```

Comment:
Just curious, why can't we use `MATCH_FIELDS` instead?

Reply:
`MATCH_FIELDS` contains `ec_number`, which does not contain letters. So to avoid unnecessary computations / creating more indexes, we don't want it to be compared in lower case (without an extra index, it would make a search for `ec_numbers` ~4 times slower (on my computer)).

Reply:
Ok! I'm just thinking that on line 26 we should be able to use `MATCH_FIELDS` directly since `field === 'ec_number'` is checked on line 22 right? So one could probably add an if condition to line 78 as well and then we wouldn't need `CASED_FIELDS`. But maybe it's cleaner keeping things as is, up to you!

Reply:
@e0 @nanjiangshu Any thoughts? :)

Reply:
I think it's cleaner to separate them.